docs/.wordlist.txt

```diff
@@ -0,0 +1,49 @@
+addons
+API
+APIs
+balancer
+Charmhub
+CLI
+Diátaxis
+dropdown
+EBS
+EKS
+enablement
+favicon
+Furo
+Git
+GitHub
+Grafana
+IAM
+installable
+JSON
+Juju
+Kubeflow
+Kubernetes
+Launchpad
+LTS
+Makefile
+Matrix
+Mattermost
+MyST
+namespace
+namespaces
+NodePort
+observability
+OEM
+OLM
+Permalink
+pre
+Quickstart
+ReadMe
+reST
+reStructuredText
+RTD
+subdirectories
+subfolders
+subtree
+Ubuntu
+UI
+UUID
+VM
+YAML
```

Comment:
We'll need our stuff here... I wonder what's the best way to address this, make a jira sub-task or a github issue?